.circleci/config.yml

```diff
@@ -15,14 +15,12 @@ jobs:
       - checkout
 
       - restore_cache:
-          key: kafkamon-{{ .Branch }}
-          key: kafkamon-master
+          key: kafkamon-{{ checksum "mix.lock" }}
```

Comment:
for some reason CircleCI doesn't like having multiple `key`s in `restore_cache` anymore
([see this](https://app.circleci.com/pipelines/github/avvo/kafkamon/5/workflows/f62e390f-053d-4ad6-ac37-043f9577b549/jobs/117))